package.json

```diff
@@ -75,19 +74,20 @@
     "eslint-import-resolver-webpack": "~0.13.2",
     "eslint-plugin-import": "~2.25.3",
     "eslint-plugin-react": "~7.27.0",
-    "extract-text-webpack-plugin": "~3.0.2",
     "history": "~5.1.0",
     "html-webpack-plugin": "~5.5.0",
     "http-server": "~14.0.0",
     "jest": "~27.3.1",
+    "mini-css-extract-plugin": "^2.4.5",
     "prop-types": "~15.7.2",
     "react": "~17.0.2",
     "react-dom": "~17.0.2",
     "react-router-dom": "~6.0.2",
     "react-test-renderer": "~17.0.2",
     "rimraf": "~3.0.0",
     "style-loader": "~3.3.1",
-    "webpack": "~5.64.1"
+    "webpack": "5.64.4",
+    "webpack-cli": "^4.9.1"
```

Comment:
`webpack-cli` is the `webpack` command, it's been split out of the main `webpack` package.